core/frontend/src/tile/GltfReader.ts

```diff
@@ -1795,6 +1865,34 @@ export abstract class GltfReader {
   }
 
   protected async resolveResources(): Promise<void> {
+    // Load any external images and buffers.
+    await this._resolveResources();
+
+    // If any meshes are draco-compressed, dynamically load the decoder module and then decode the meshes.
+    const dracoMeshes: DracoMeshCompression[] = [];
+
+    for (const node of this.traverseScene()) {
+      for (const meshId of getNodeMeshIds(node)) {
+        const mesh = this._meshes[meshId];
+        if (mesh?.primitives)
+          for (const primitive of mesh.primitives)
+            if (primitive.extensions?.KHR_draco_mesh_compression)
+              dracoMeshes.push(primitive.extensions.KHR_draco_mesh_compression);
+      }
+    }
+
+    if (dracoMeshes.length === 0)
+      return;
+
+    try {
+      const dracoLoader = (await import("@loaders.gl/draco")).DracoLoader;
+      await Promise.all(dracoMeshes.map(async (x) => this.decodeDracoMesh(x, dracoLoader)));
+    } catch (_) {
+      //
```

Comment:
Log?

Reply:
We are really bad about logging on frontend. I actually don't know where those log messages end up. But I will add logging.

Reply:
Added logging. I don't like having two logging statements, but there doesn't appear to be a way to log the exception (with potentially very useful information) along with a description of what went wrong, in a single logger call.
It appears nobody uses Logger.logException, which is a shame - most of our log messages end up being vague and unhelpful.